extensions/DPLforum/DPLforum_body.php

```diff
@@ -377,25 +377,38 @@ function parse( &$input, Parser $parser ) {
 					$first_time = '';
 				}
 
-				if( isset( $row->first_user ) ) {
-					$first_user = $row->first_user;
-				} else {
-					$first_user = '';
+				$first_user = '';
+				if( isset( $row->first_user ) || isset( $row->first_user_text )) {
+					$first_user = User::getUsername( $row->first_user, $row->first_user_text );
```

Comment:
We must make sure both arguments to this method are defined (probably they are) due to strict type checks